examples/router/src/pages/post_list.rs

```diff
@@ -92,4 +82,11 @@ impl PostList {
             </div>
         }
     }
+
+    fn current_page(&self) -> u64 {
+        service::query()
+            .get("page")
+            .and_then(|it| it.parse().ok())
+            .unwrap_or(1)
```

Comment:
I think we should let `yew_router::utils::get_query_params()` expose what it does and `yew_router::parse_query` will handle the conversion using `serde`. How does that sound?

Reply:
Well, I went ahead and did it.

Reply:
yeah, that's what I had in mind. You might even want to use `serde` for the `yew_router::utils::get_query_params` (it supports deserializing to a hashmap) because that *should* be more performant (since it removes a bunch of calls to JS) and greatly simplifies the code.

Reply:
That sounds like a good idea. The thing preventing me is how I should obtain everything `?` from the URL. Will a simple split work fine?

Reply:
Just use [URL.search](https://developer.mozilla.org/en-US/docs/Web/API/URL/search) instead of `searchParams` and strip the leading '?'.
The result will be an empty string if there is no query string at all.

Reply:
Used that, it works.